sourcemash/api/items.py

```diff
@@ -98,6 +103,7 @@ def put(self, id):
         # Toggle saved-for-later status (aka bookmarked)
         if args.saved != None:
             user_item.saved = args.saved
+            user_item.last_modified = datetime.utcnow()
```

Comment:
But do we want it on an unread change too? It would make sense for the answer to that question to be yes.